extensions/gitpod-remote/src/extension.ts

```diff
@@ -163,6 +163,9 @@ export function registerHearbeat(context: GitpodExtensionContext): void {
 		}
 		try {
 			await context.gitpod.server.sendHeartBeat({ instanceId: context.info.getInstanceId(), wasClosed });
+			if (wasClosed) {
+				await context.fireAnalyticsEvent({ eventName: 'ide_close_signal', properties: { clientKind: 'vscode' } }).catch(() => { });
```

Comment:
not sure whether await is necessary, it also look like we swallow an error?

How do we do it in other places?

Reply:
I don't know what happens when you do this in vscode, but if we have an error in node if there is no catch, we have a warning, and on higher versions we even have an error

Reply:
If we don't add await and catch this is an `Unhandled promise rejection` If we add await then the error can be caught but we don't actually want to print the error, it's only mean send analyze failed, not mean send heartbeat failed

Reply:
For the sake of consistency in code style, I modified it to be no await no catch